### DISCUSSION TITLE
Has Minidexed outgrown default performance.ini?

### DISCUSSION THREAD

**Discussion by miotislucifugis** [probonopd/MiniDexed#715] [Ideas]
While ive been a minidexed user for a while now, Ive only recently started exploring the newer performance features (and the rather excellent default performances. great work!)         One thing that Im finding a bit awkward and archaic is the role/ use of  default performance.ini.      Now that we have a system of organizing performance presets into banks,  couldnt minidexed just simply have a default bank/performance number rather than a separate performance file that exists outside of the normal bank/performance structure?   

As it is now,  even if you assign a new default performance,  when minidexed boots, it is always essentially starting on bank 1, preset "0"  and if the rest of the patches that you want to uses are in (for example) bank 3, presets 100-120, it requires a bit of tedious and necessary navigation   (a specific program change number, Loooots of program up/down button click/ or menu navigation.)     When saving the "default" performance, couldnt minidexed just as easily save "bank= 3, performance= 100", and load that bank and performance (and starting at that position in the preset list)  on boot up?

**Comment by Banana71** [comment]
Hello Charles, 
the management of performances has grown over time and is not entirely logical.
The way it is now with the separate default performance.ini in the root directory is, I think, unique.

I know of many possibilities from other synthesizers:
- The instrument starts with the first performance 001:001 Bank 1, Performance 1)
- The performance can be freely selected and loaded when the instrument is started.
- The instrument loads the performance that was selected in the last session.
- .....

In my opinion, all performances should be in the /performance directory. And the behavior when starting the miniDexed is defined in a global.ini, for example.
This global.ini should be configured by the UI of the miniDexed and contain a few other parameters that can be changed by the user.

But who is supposed to change and program all of this?
